utils/build_swift/postbuild_steps.py

```diff
@@ -0,0 +1,115 @@
+#===--- postbuild.py -----------------------------------------------------===#
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https:#swift.org/LICENSE.txt for license information
+# See https:#swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+#===----------------------------------------------------------------------===#
+
+import os
+import platform
+import shutil
+
+import swift_build_support.swift_build_support.products as products
+import swift_build_support.swift_build_support.tar as tar
+import swift_build_support.swift_build_support.shell as shell
+from swift_build_support.swift_build_support.targets \
+    import StdlibDeploymentTarget
+
+def create_symbols_package(args):
+    assert args.symbols_package
+    print('--- Creating symbols package ---')
+    print('-- Package file: {} --'.format(args.symbols_package))
+
+    if platform.system() == 'Darwin':
+        prefix = targets.darwin_toolchain_prefix(args.install_prefix)
+    else:
+        prefix = args.install_prefix
+
+    # As a security measure, `tar` normally strips leading '/' from paths
+    # it is archiving. To stay safe, we change working directories, then
+    # run `tar` without the leading '/' (we remove it ourselves to keep
+    # `tar` from emitting a warning).
+    with shell.pushd(args.install_symroot):
+        tar.tar(source=prefix.lstrip('/'),
+                destination=args.symbols_package)
+
+def build_benchmarks_linux(invocation):
+    # Perform some sanity checking.
+    args = invocation.args
+
+    # For Linux, we need a full install root.
+    assert args.install_destdir, "Can only benchmark if we compile a dest root?!"
+    # We also need to have built and installed Foundation into the destdir.
+    assert args.install_foundation, "Must build/install foundation to do benchmarks."
+
+    install_destdir = os.path.join(args.install_destdir,
+                                   args.install_prefix.lstrip('/'))
+
+    workspace = invocation.workspace
+    llvm = products.LLVM
```

Comment:
Why not just import `LLVM` and `Swift` from products directly at the top?

`from swift_submission_support.swift_submission_support.products import LLVM, Swift`